org.civicrm.reqangular/src/common/modules/directives.js

```diff
@@ -2,18 +2,21 @@ define([
   'common/angular',
   'common/decorators/ui-select/ui-select-choices',
   'common/decorators/angular-date/datepicker-addon',
+  'common/decorators/ui-bootstrap/uib-tabset',
+  'common/angularBootstrap',
   'common/ui-select',
   'common/modules/templates',
   'common/modules/controllers',
   'common/modules/services',
   'common/modules/apis'
-], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon) {
+], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon, uibTabset) {
```

Comment:
> I think it would be better to split this module in two, common.directives and common.decorators

Not sure if this is a good idea. Because, `common.directives` is importing all the directives , so If we want to override some of them it is better to do there itself? The idea is `common.directives` imports all common directives and overrides them before exporting the same.